climada_petals/hazard/test/test_wildfire.py

```diff
@@ -52,7 +52,8 @@ def def_ori_centroids(firms, centr_res_factor):
         firms['longitude'].min() : firms['longitude'].max() : complex(0, nb_centr_lon)]). \
         reshape(2, nb_centr_lat*nb_centr_lon).transpose()
 
-    centroids = Centroids.from_lat_lon(coord[:, 0], coord[:, 1])
+    centroids = Centroids()
+    centroids.set_lat_lon(coord[:, 0], coord[:, 1])
```

Comment:
```suggestion
    centroids = Centroids.from_lat_lon(coord[:, 0], coord[:, 1])
```
Why did you change this back to the old set method? The new way is as was.

Reply:
whoop sorry, I think I messed something up with the climada on my laptop =)